tripal_chado/migration/export_tripal3_entity_mapping.php

```diff
@@ -16,7 +16,7 @@ function find_all_published(string $output_file_name) {
 
   // Get a list of defined bio_data bundles
   $sql = "SELECT name, label FROM [tripal_bundle] ORDER BY term_id";
-  $results = chado_query($sql, []);
+  $results = \Drupal::service('tripal_chado.database')->query($sql, []);
```

Comment:
```suggestion
  $results = chado_query($sql, []);
```
This is a special case, this script is designed to be run on a Tripal 3 site, so it won't have the new service!